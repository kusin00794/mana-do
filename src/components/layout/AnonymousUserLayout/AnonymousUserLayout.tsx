import React, { FC } from 'react';

import '../../../styles/common.scss';

const AnonymousUserLayout: FC = ({ children }) => {
  return <div className="layout logged-in-layout">{children}</div>;
};

export default AnonymousUserLayout;
