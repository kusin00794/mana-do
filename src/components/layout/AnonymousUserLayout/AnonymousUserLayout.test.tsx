import React from 'react';
import { BrowserRouter } from 'react-router-dom';

import AnonymousUserLayout from './AnonymousUserLayout';
import { testSnapshots } from '../../../utils/test';

const Wrapper = () => (
  <BrowserRouter>
    <AnonymousUserLayout>
      <div>Mocked Component</div>
    </AnonymousUserLayout>
  </BrowserRouter>
);

describe('Snapshot for AnonymousUserLayout', () => {
  testSnapshots(Wrapper, [
    {
      props: {},
      description: 'render AnonymousUserLayout',
    },
  ]);
});
