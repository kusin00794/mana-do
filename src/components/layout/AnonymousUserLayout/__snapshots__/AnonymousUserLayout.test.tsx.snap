// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for AnonymousUserLayout snapshots render AnonymousUserLayout 1`] = `
<div
  className="layout logged-in-layout"
>
  <div>
    Mocked Component
  </div>
</div>
`;
